_basics/install_platformio.md

```diff
@@ -156,12 +156,20 @@ This step is optional, but we recommend doing the minimal build because configur
        The project folder and its contents should appear in the Project Navigator on the left side.
 
        ![Explorer](/assets/images/basics/install_platformio/explorer.png)
+       
+### Prepare **platformio.ini**
+
+1. Open the **platformio.ini** file by clicking on it.
+
+2. Use the **Find** command or scroll down to locate the entry for **env_default** and make sure its set correctly for your board. (e.g., **megaatmeg2560** or **LPC1768**).
```

Comment:
This is optional. The `env_default` setting doesn't have to be set to your platform.

Reply:
but would it not help if env_default is set correct  if you use vscode+platformio?